Hotfix/add correct operational class to steering request

kantera800 opened the pull request:
up until now BSS_TM_REQ was sent with  operational class = 0
 This is wrong and some clients reject steering requests because of that.

Add oper_class parameter to function signature and use in BSS_TM_REQ message .

tomereli commented on agent/src/beerocks/slave/ap_manager_thread.cpp:
```diff
@@ -808,7 +808,8 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
                    << " to bssid = " << target_bssid
                    << " channel = " << int(request->params().target.channel);
-        ap_wlan_hal->sta_bss_steer(sta_mac, target_bssid, request->params().target.channel,
+        ap_wlan_hal->sta_bss_steer(sta_mac, target_bssid, request->params().target.operating_class,
```
Why can't this be squashed to the previous commit?

kantera800 replied:
sure , just after the 20 commits in my last pr i felt i had to split something :)

arnout reviewed (approved):
The merge commit shouldn't be there, of course.

It's funny that certification test 4.8.1 passes even without this change. I guess the CTT STA doesn't even look at the operating class...

Commit:
bwl: add operational class to BSS_TM_REQ

Up until now BSS_TM_REQ was sent with operational class 0.
This is wrong and clients could reject steering requests because of
this.
Add oper_class parameter to function signature.
Operational class parameter is already sent in the TLV
so just pass it to the sta_bss_steer and  use in BSS_TM_REQ message.

Signed-off-by: Alex Kanter <alexx.kanter@intel.com>

Commit:
Merge branch 'master' into hotfix/add-correct-operational-class-to-steering-request
